docs/reference/setup/install/docker.asciidoc

```diff
@@ -202,7 +203,17 @@ endif::[]
 To stop the cluster, type `docker-compose down`. Data volumes will persist, so it's possible to start the cluster again with the same data using `docker-compose up`.
 To destroy the cluster **and the data volumes** just type `docker-compose down -v`.
 
-===== Inspect status of cluster:
+### Inspect status of cluster:
+
+#### Via Browser
+
+**url:** http://http://docker.local:9200/
```

Comment:
This URL didn't work on my Windows 10 workstation (with Docker for Mac).
Instead I found that the usual `http://127.0.0.1:9200/` worked which renders nicely.

This would need to be reformatted for asciidoc.

I spent a bit of time and ended up with the following asciidoc excerpt which looks nice.
I am wondering if suggesting installing curl on windows (https://skanthak.homepage.t-online.de/curl.html) would also be a good advice as this is handy to have.

```asciidoc

===== Inspect status of cluster:

. Via Browser
+
http://127.0.0.1:9200/
+
**username:** elastic
+
**password:** changeme
+
. Via Curl
+
["source","sh"]
--------------------------------------------
curl -u elastic http://127.0.0.1:9200/_cat/health
Enter host password for user 'elastic':
1472225929 15:38:49 docker-cluster green 2 2 4 2 0 0 0 0 - 100.0%
--------------------------------------------
// NOTCONSOLE
// This is demonstrating curl. Console will prompt you for a username and
// password so no need to demonstrate that. Converting this would not show the
// important `-u elastic` parameters for `curl`.

```